README.md

```diff
@@ -76,8 +78,10 @@ git clone github.com/Muhammad0602/Portfolio-mobile-first.git
 
 ### Usage
 
-To run the project, execute the following command:
- open the project by launching the Live Server plugging in visual studio code(or your code editor) 
+To run the project, execute the following commands:
+ - first `npm install`
+ - second `npm run build`
+ - and lastly `npm start` 
 
```

Comment:
- [ ] Let's add the testing instructions in your readme file. I tried `jest` commands, but it did not work.